CONTRIBUTING.md

```diff
@@ -2,14 +2,15 @@
 - [First Things First](#first-things-first)
 - [Ways to Contribute](#ways-to-contribute)
   - [Bug Reports](#bug-reports)
-  - [Feature Requests](#feature-requests)
+  - [Feature Requests \& Proposals](#feature-requests--proposals)
```

Comment:
TOC doesn't match, update.

Reply:
This change actually fixes the TOC for the Feature requests heading. The previous link does not actually work. It was my markdown linter that actually caught this error

Reply:
Looked at @ashwin-pc's forks seem to work as expected. [[link](https://github.com/ashwin-pc/opensearch.github/blob/doc/feature-branching/CONTRIBUTING.md#feature-requests--proposals)]